### ISSUE TITLE
Systemd on Debian Buster complains about init.d script

### ISSUE THREAD

**Issue by eblosug** [eblocker/eblocker-dns#1]
Systemd on Debian Buster complains that the PID file is not owned by root:
```
root@eblocker:/opt/eblocker-dns/log# systemctl status eblocker-dns
● eblocker-dns.service - LSB: eblocker-dns initscript
   Loaded: loaded (/etc/init.d/eblocker-dns; generated)
   Active: failed (Result: protocol) since Mon 2020-02-03 17:33:04 CET; 2s ago
     Docs: man:systemd-sysv-generator(8)
  Process: 982 ExecStart=/etc/init.d/eblocker-dns start (code=exited, status=0/SUCCESS)

Feb 03 17:33:00 eblocker systemd[1]: Starting LSB: eblocker-dns initscript...
Feb 03 17:33:00 eblocker su[983]: (to eblocker-dns) root on none
Feb 03 17:33:00 eblocker su[983]: pam_unix(su:session): session opened for user eblocker-dns by (uid=0)
Feb 03 17:33:04 eblocker eblocker-dns[982]: Starting eblocker-dns daemon...
Feb 03 17:33:04 eblocker eblocker-dns[982]: Daemon status: running pid=994
Feb 03 17:33:04 eblocker systemd[1]: eblocker-dns.service: New main PID 994 does not belong to service, and PID file is not owned by root. Refusing.
Feb 03 17:33:04 eblocker systemd[1]: eblocker-dns.service: New main PID 994 does not belong to service, and PID file is not owned by root. Refusing.
Feb 03 17:33:04 eblocker systemd[1]: eblocker-dns.service: Failed with result 'protocol'.
Feb 03 17:33:04 eblocker systemd[1]: Failed to start LSB: eblocker-dns initscript.
```
The start script runs the DNS ruby server as user `eblocker-dns`: https://github.com/eblocker/eblocker-dns/blob/develop/etc/init.d/eblocker-dns

**Comment by eblosug** [comment]
There are some good explanations in this article: https://unix.stackexchange.com/a/534962

**Comment by eblosug** [comment]
We should probably retire the `init.d` script and write a proper systemd service file.

**Comment by eblosug** [comment]
Fixed in 2.4.0
